inst/config/tplyr_shift.yaml

```diff
@@ -1,6 +1,6 @@
 label: Shift Table {Tplyr} + {kable}
 type: html
 domain: multiple
-pacakage: Tplyr
+package: safetyCharts
 workflow:
```

Comment:
Does this yaml need the export?

Reply:
Doesn't hurt to add it, but will default to TRUE if nothing is provided.